numpy/typing/_array_like.py

```diff
@@ -16,12 +18,15 @@
     else:
         HAVE_PROTOCOL = True
 
+_DType = TypeVar("_DType", bound="dtype[Any]")
+
 if TYPE_CHECKING or HAVE_PROTOCOL:
-    class _SupportsArray(Protocol):
-        @overload
-        def __array__(self, __dtype: DTypeLike = ...) -> ndarray: ...
-        @overload
-        def __array__(self, dtype: DTypeLike = ...) -> ndarray: ...
+    # The `_SupportsArray` protocol only cares about the default dtype
+    # (i.e. `dtype=None`) ofthe to-be returned array.
```

Comment:
typo "ofthe".

Reply:
Good call, fixed as of https://github.com/numpy/numpy/pull/17981/commits/6139ed42af271d37234320e83a41000d93bdeae1.